configure.ac

```diff
@@ -139,6 +139,11 @@ AC_ARG_ENABLE(module_musig,
     [enable_module_musig=$enableval],
     [enable_module_musig=no])
 
+AC_ARG_ENABLE(module_threshold,
```

Comment:
Maybe call the module thresholdsig for consistency with musig and with the api names?